Bottom: ae602d7325b986d4fb2fbda42e09eeeb2d54ebb3
Top:    bb9ef433599ef5f4db95e5c3c50c3b2c87c005f4
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-20 09:06:24 -0500

Refresh of nexus-haproxy

---
diff --git a/ansible-playbook/group_vars/all/nexus3.yml b/ansible-playbook/group_vars/all/nexus3.yml
index a93dfc53a..5963e9fa3 100644
--- a/ansible-playbook/group_vars/all/nexus3.yml
+++ b/ansible-playbook/group_vars/all/nexus3.yml
@@ -4,7 +4,7 @@ nexus_networks:
     aliases:
       - '{{ nexus_hostname }}'
 docker: true
-nexus_default_port: 80
+nexus_default_port: 8081
 
 nexus_email_server_enabled: true
 nexus_email_server_host: "mail.novatech-llc.com"
diff --git a/roles/nexus3/tasks/nexus_install_docker.yml b/roles/nexus3/tasks/nexus_install_docker.yml
index 54a42c1a2..26bf93017 100644
--- a/roles/nexus3/tasks/nexus_install_docker.yml
+++ b/roles/nexus3/tasks/nexus_install_docker.yml
@@ -70,6 +70,7 @@
     networks: '{{ nexus_networks }}'
     volumes: '{{ nexus_volumes }}'
     ports: '{{ nexus_port_args }}'
+    purge_networks: true
     state: started
   register: nexus_container
 
@@ -84,8 +85,8 @@
     uri: "https://{{nexus_hostname}}{{nexus_default_context_path}}"
     private_uri: "http://{{host}}:{{port}}{{nexus_default_context_path}}"
   vars:
-    host: "{{ nexus_container.ansible_facts.docker_container.NetworkSettings.Ports['8081/tcp'][0].HostIp }}"
-    port: "{{ nexus_container.ansible_facts.docker_container.NetworkSettings.Ports['8081/tcp'][0].HostPort }}"
+    host: "{{ nexus_container.ansible_facts.docker_container.NetworkSettings.Networks[docker_network_frontend].IPAddress }}"
+    port: "{{ nexus_default_port }}"
 
 - name: First-time install admin password
   set_fact:
@@ -108,7 +109,7 @@
     status_code: 200,204
     user: 'admin'
   register: nexus_ping
-  retries: 20
+  retries: 3
   delay: 5
   until: nexus_ping.content == "pong\n"
